src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,35 @@
+.form-field {
+  box-sizing: border-box;
+  display: block;
+  height: 48px;
+  width: 100%;
+  margin: 0 0 16px;
+  padding: 14px 16px;
+  font-size: 14px;
+  line-height: 20px;
+  color: $colorBlack;
+  background-color: $formBackgroundColor;
+  border-radius: 40px;
+  border: 1px solid transparent;
+
+  &::placeholder {
+    font-size: 14px;
+    line-height: 20px;
+    color: $formTextColor;
+  }
+
+  &:hover {
+    border: 1px solid #d5d7de;
+  }
+
+  &:focus {
+    border: 1px solid $colorBlack;
+  }
+
+  &--message {
+    margin: 0 0 32px;
+    min-height: 148px;
+    font-family: inherit;
```

Comment:
You may want to remove the 'font-family: inherit;' declaration since it's unnecessary. The font family will be inherited by default. 🧹